alerts.js

```diff
@@ -0,0 +1,14 @@
+
+// ruleid:javascript-prompt
+var name = prompt('what is your name');
```

Comment:
:exclamation:**Cycode**: SAST violation: 'Found prompt() call'.
 
found prompt() call; should this be in production code?
 
Severity: Medium
 <!-- 9d443cd2-d992-4b06-aa76-262fe00e36a8 -->